daffodil-lib/src/main/scala/org/apache/daffodil/util/MStack.scala

```diff
@@ -182,7 +182,7 @@ object MStackOfAnyRef {
  * things.
  */
 protected abstract class MStack[@specialized T] private[util] (
-  arrayAllocator: (Int) => Array[T], nullValue: T) extends Serializable {
+  arrayAllocator: (Int) => Array[T], nullValue: T) {
```

Comment:
It was no longer needed since I ended up switching data structures.  The Serializable here was only added for Variables during the newVariableInstance redesign, so it is no longer necessary.